src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,12 @@
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+}
+
+body,
+p {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", Arial, sans-serif;
```

Comment:
```suggestion
```

Reply:
I left Arial font because according to previous tasks I was recommended to set an alternative font in case the first one is not available